src/plugins/preauth/pkinit/pkinit_constants.c

```diff
@@ -320,6 +320,33 @@ static const uint8_t o4096[] = {
     0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF
 };
 
+/* Named curve prime256v1 (1.2.840.10045.3.1.7) as parameters for RFC 3279
+ * section 2.3.5 id-ecPublicKey */
+static const uint8_t p256[] = {
+    0x06, 0x08, 0x2A, 0x86, 0x48, 0xCE, 0x3D, 0x03, 0x01, 0x07
+};
+
+/* Named curve secp384r1 (1.3.132.0.34) as parameters for RFC 3279 section
+ * 2.3.5 id-ecPublicKey */
+static const uint8_t p384[] = {
+    0x06, 0x05, 0x2B, 0x81, 0x04, 0x00, 0x22
+};
+
+/* Named curve secp521r1 (1.3.132.0.35) as parameters for RFC 3279 section
+ * 2.3.5 id-ecPublicKey */
+static const uint8_t p521[] = {
+    0x06, 0x05, 0x2B, 0x81, 0x04, 0x00, 0x23
+};
```

Comment:
These 2 curves are actually defined in [RFC5480](https://datatracker.ietf.org/doc/html/rfc5480#section-2.1.1.1). Only `prime256v1` is defined in [RFC3279](https://datatracker.ietf.org/doc/html/rfc3279#page-23)

Reply:
I'll add an RFC 5480 reference.  They are still here as parameters for id-ecPublicKey from RFC 3279 (which just means they have a DER OBJECT IDENTIFIER tag and length rather than being the untagged encoding).